service.libraryautoupdate/addon.xml

```diff
@@ -0,0 +1,88 @@
+    <description lang="pt_BR">Este é um serviço Kodi que irá atualizar a sua coleção de vídeos e músicas através de agendador de atividade. Você pode selecionar um intervalo diferente para escanear seus bancos de dados de mídia (áudio, vídeo, tanto) ou você pode definir um temporizador estilo cron  para maior controle. Se você estiver reproduzindo um arquivo de áudio ou vídeo quando o cronômetro iniciar, ele pode pular o processo de atualização da biblioteca até que seja concluída, de modo que sua experiência de mídia não é interrompido. Atualizar um caminho de vídeo específico, e de limpeza das bibliotecas de música / vídeo é agora suportado.</description>
+    <description lang="ru_RU">Это сервис, который обновляет вашу медиатеку по таймеру. Вы можете выбрать разные периоды для сканирования  ваших баз (Аудио, Видео, и то и то) или установить таймер по-времени для большего контроля. Если запущено проигрывание аудио или видео файла, когда срабатывает таймер, он может отменить процесс обновления до тех пор, пока Вы не закончите прослушивание или просмотр чего-либо. &#10;Выборочное обновление папок с Видео, очистка медиатеки теперь также поддерживается.</description>
+    <description lang="sk_SK">Toto je Kodi služba, ktorá aktualizuje Vašu Kodi Hudobnú a Video knižnicu podľa časovača. Môžete zvoliť rôzne intervaly aktualizácie databáz (Hudba,Video,oboje) alebo pre väčšiu kontrolu zadáte časovač podobný Cron-u. Pokiaľ práve prehrávate hudobný alebo video súbor počas štartu časovača, môže byť aktualizácia odložená, aby nebol Váš mediálny zážitok narušený. Aktualizácia špecifických ciest a upratovanie Hudobnej/Video databázy je tiež podporované. </description>
+    <description lang="sv_SE">Detta är en Kodi-service som schemalagt kommer att uppdatera ditt musik och videobibliotek. Du kan välja olika intervall att skanna dina mediadatabaser (Musik,Video,Båda) eller så kan du ange ett cron-inspirerad schemaläggare för bättre kontroll. Om du spelare en musik eller videofil när schemaläggaren startar, kan det hoppa över biblioteksuppdateringen till uppspelningen är klar så mediaupplevelsen inte avbryts. Uppdatering av en specifik videosökväg och rensning av musik/videobiblioteken stödjs nu.</description>
+    <description lang="zh_CN">这是一个定时更新你的音乐和视频资料库的Kodi服务程序。你可以设置不同的时间间隔来扫描媒体库（音频、视频或二者），或者设置一个cron类型的定时器来获得更好的控制。如果定时动作开始时你正在播放一个音频或视频文件，它会跳过此次更新直到播放完毕以避免影响你欣赏媒体。现在已支持更新指定的视频路径以及清理音视频资料库。</description>
+    <language></language>
+    <license>The MIT License</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=119520</forum>
+    <source>https://github.com/robweber/xbmclibraryautoupdate</source>
+	<assets>
+		<icon>resources/media/icon.png</icon>
+	</assets>
```

Comment:
I guess it also needs a fanart since the add-on is an executable

Reply:
Does it **need** it? Travis Buddy output says "recommended". I can probably dig some up but I'm not really a graphic design kind of guy.

Reply:
It's fine